### ISSUE TITLE
Is this project abandoned? I am happy to take over

### ISSUE THREAD

**Issue by alexbacchin-asx** [mercari/tfnotify#127]
## WHAT

(Write what you need)

## WHY

(Write the background of this issue)

**Comment by alexbacchin-asx** [comment]
@suzuki-shunsuke Thanks for sharing, it looks very promising. Doe sit support Gitlab?

**Comment by suzuki-shunsuke** [comment]
No. tfcmt doesn't support GitLab, but [tfcmt-gitlab](https://github.com/hirosassa/tfcmt-gitlab), which is a fork of tfcmt for GitLab, supports GitLab.
